common/test/acceptance/tests/lms/test_progress_page.py

```diff
@@ -125,6 +125,7 @@ def _logged_in_session(self, staff=False):
 
 
 @attr(shard=9)
+@flaky(max_runs=20, min_passes=20)
```

Comment:
Not merging this, just want to validate the bok choy fixes I made yesterday, since I came across some slight timing issues when running locally.